File: app/src/main/java/com/example/zentask/TaskLogic/Task.java
package com.example.zentask.TaskLogic;

public class Task {
    public String name;
    public String date;

    public int time;
    
    public String ampm;
    public String description;
    public boolean isArchived;

    public Task(String name, String date, int time, String ampm, String desc) {
        this.name = name;
        this.date = date;
        this.time = time;
        this.ampm = ampm;
        this.description = desc;
        this.isArchived = false;
    }
}

File: app/src/main/java/com/example/zentask/TaskLogic/Task.kt
package com.example.zentask.TaskLogic

import java.util.UUID

data class Task(
    var name: String,
    var date: String,
    var time: Int,
    var ampm: String,
    var description: String,
    var isArchived: Boolean = false,
    val id: String = UUID.randomUUID().toString()
)